packages/nimble-blazor/NimbleBlazor/DateTextColumnTypes.cs

```diff
@@ -4,12 +4,17 @@ namespace NimbleBlazor;
 
 public enum DateTextFormat
 {
+    Default,
     Custom
 }
 
 internal static class DateTextFormatExtensions
 {
-    private static readonly Dictionary<DateTextFormat, string> _enumValues = AttributeHelpers.GetEnumNamesAsKebabCaseValues<DateTextFormat>();
+    private static readonly Dictionary<DateTextFormat, string?> _enumValues = new Dictionary<DateTextFormat, string?>
```

Comment:
This should have a consistent implementation as our other enums in Blazor.

A few examples:
- [SpinnerAppearance](https://github.com/ni/nimble/blob/main/packages/nimble-blazor/NimbleBlazor/SpinnerAppearance.cs)
- [AnchorAppearance](https://github.com/ni/nimble/blob/main/packages/nimble-blazor/NimbleBlazor/AnchorAppearance.cs)
- [BannerSeverity](https://github.com/ni/nimble/blob/main/packages/nimble-blazor/NimbleBlazor/BannerSeverity.cs)